### PR TITLE
[typing] Kill `Decimal.Value` method

### PR DESCRIPTION
**Pull request by nathan-artie**

### CODE REVIEW

File: clients/shared/utils.go [hunk 1]
```diff
@@ -44,8 +44,7 @@ func DefaultValue(column columns.Column, dialect sql.Dialect, additionalDateFmts
 			return nil, fmt.Errorf("colVal is not type *decimal.Decimal")
 		}
 
-		// TODO: Call [String] instead.
-		return val.Value(), nil
+		return val.String(), nil
```
Comment (nathan-artie) [inline clients/shared/utils.go:47]: This is the only place it was used.

Comment (nathan-artie) [reply clients/shared/utils.go:47]: And this function is only called when backfilling.

Comment (nathan-artie) [reply clients/shared/utils.go:47]: @Tang8330 I feel like since we were sometimes emitting a string before, always emitting a string should be fine?

Comment (Tang8330) [reply clients/shared/utils.go:47]: Yeah it's fine, I just tested it

**Review by Tang8330 — APPROVED** [review]

File: clients/shared/utils.go (hunk 1 above)
Comment (nathan-artie) [reply clients/shared/utils.go:47]: Sweet thanks!

**Pull request merged**